README.md

```diff
@@ -67,6 +67,7 @@ Please read the reference documentation
 and the paper's Supplement (which includes a tutorial) before using the
 application. The documents are available at:
 
+- https://gmd.copernicus.org/articles/15/8983/2022/gmd-15-8983-2022.pdf
```

Comment:
This is incorrect.

This is your addition

```
Publications
============

Please read the reference documentation

   Rautenhaus, M., Bauer, G., and Doernbrack, A.: A web service based
   tool to plan atmospheric research flights, Geosci. Model Dev., 5,
   55-71, doi:10.5194/gmd-5-55-2012, 2012.

and the paper's Supplement (which includes a tutorial) before using the
application. The documents are available at:

- https://gmd.copernicus.org/articles/15/8983/2022/gmd-15-8983-2022.pdf
- http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf
- http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012-supplement.pdf

For copyright information, please see the files NOTICE and LICENSE, located
in the same directory as this README file.
```

The new paper is not mentioned. The link added does not belong to Marcs publication.

In principle that is quite similar as in the about or NOTICE, it is just better readable, without starting the UI.


I prepared a diff, how it should be
[readme.diff.txt](https://github.com/Open-MSS/MSS/files/10879335/readme.diff.txt)